packages/providers/src/validation.ts

```diff
@@ -200,6 +234,8 @@ export const RpcReturnTypesUnion = z.discriminatedUnion("method", [
 	SetStateRequestReturnSchema,
 	GetStateRequestReturnSchema,
 	StorePrivateCredentialReturnSchema,
+	PresentationRequestReturnSchema,
+	signFieldsWithPassphraseRequestReturnSchema,
```

Comment:
We should remove `signFieldsWithPassphrase` from the RPC type since it's I'd say "nice to have" and it shouldn't be a requirement for all the wallets to implement 🤔